src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -905,7 +907,8 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
         )
       }
 
-    def isStrictFP: Boolean    = !isDeferred && (hasAnnotation(ScalaStrictFPAttr) || originalOwner.isStrictFP)
+    @tailrec
+    final def isStrictFP: Boolean    = this != NoSymbol && !isDeferred && (hasAnnotation(ScalaStrictFPAttr) || originalOwner.isStrictFP)
```

Comment:
Removed override in `NoSymbol` to make this self-contained and final.

Reply:
TIL.

Reply:
(I meant about the change in receiver.)